packages/components/src/timepicker/src/components/osds-timepicker/core/controller.ts

```diff
@@ -0,0 +1,62 @@
+import type { ODS_TIMEZONE } from '../constants/timezones';
+import type { OsdsTimepicker } from '../osds-timepicker';
+import { ODS_TIMEZONES } from '../constants/timezones';
+import { ODS_TIMEZONES_PRESET } from '../constants/timezones-preset';
+
+class OdsTimepickerController {
+
+  constructor(private readonly component: OsdsTimepicker) {
+  }
+
+  formatValue(value: string, withSeconds: boolean): string {
+    if (withSeconds === false && value.match(/:/g)?.length === 2) {
+      const inputValue = value.split(':');
+      return `${ inputValue[0] }:${ inputValue[1] }`;
+    }
+
+    if (withSeconds === true && value.match(/:/g)?.length === 1) {
+      return `${ value }:00`;
+    }
+
+    return '';
+  }
+
+  handleTimezones() :void {
+    const check = this.parseTimezones(this.getTimezonesList());
+    this.component.timezonesList = check as ODS_TIMEZONE[] ;
```

Comment:
you shouldn't have to force type as it's the actual return type of `parseTimezones`